frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -135,7 +136,9 @@ export default class AccordionList extends Component {
   };
 
   componentDidMount() {
-    this.container = ReactDOM.findDOMNode(this);
+    const container = this.isVirtualized()
+      ? this._list?.Grid?._scrollingContainer
+      : this.listRootRef.current;
```

Comment:
`findDOMNode` resolves to the grid's scrolling container if it is virtualized, otherwise it resolves to the accordion root